init.sh

```diff
@@ -43,5 +43,27 @@ fi
 tailscaled --tun=userspace-networking -state=/tailscale/tailscaled.state &
 sleep 5
 tailscale up -authkey "${TAILSCALE_KEY}" -hostname "${TAILSCALE_HOSTNAME}" $@
+IP_ADDRESS=$(tailscale ip -4)
 
+DEVICE_ID=$(curl -X GET "https://api.balena-cloud.com/v5/device?\$filter=uuid%20eq%20'${BALENA_DEVICE_UUID}'&\$select=id" -H "Content-Type: application/json" -H "Authorization: Bearer ${BALENA_API_KEY}" -s | jq -r '.d | .[] | .id')
```

Comment:
Can we make this section optional via env variables?

Reply:
I guess that's possible, but what's the intention for that? With the current change (only) the IP of the device on the Tailscale network will be added as a tag within Balena. Is it getting out of date?